bin/generate_npm_tarball.sh

```diff
@@ -6,13 +6,27 @@ if [ $# -ne 2 ]; then
 fi
 package=$1
 output=$2
-
 wd=$(mktemp -d)
 trap "rm -r '$wd'" EXIT INT TERM
 
-mkdir $wd/cache $wd/install
+create_cache() {
+  node_modules/npm/bin/npm-cli.js install --cache $wd/cache $package --production --verbose
+}
+
+mkdir $wd/cache-combined $wd/cache $wd/install
 cd $wd/install
-npm install --cache $wd/cache $package --production --verbose
+npm install npm@1.x
+create_cache
+# save el7 (which uses 1.3.6 npm) results before running it again
+find $wd/cache/registry.npmjs.org -name .cache.json > ~/workspace/npm2rpm/npm1
```

Comment:
1. For 1.3.x, this should be find $wd/cache/ only, as it doesn't use the registry.npmjs.org subdir.
2. ~/workspace doesn't exist
3. This file's unused, but I think it ought to be used with the `mv` to the cache-combined dir - instead of moving everything in registry.npmjs.org below (which doesn't exist with 1.3.x), it should perhaps do: `find $wd/cache -name .cache.json -exec mv -t $wd/cache-combined/ {} +`